### PR TITLE
chore(password): graduate confirm password experiment

### LINKED ISSUE #5854: Enable `confirm password` experiment for all users
From product meeting, we were given the green light to enable this experiment for all users.

Ref: https://projects.growthhackers.com/u/orgs/mozilla-corporation/projects/FAAS/cards/FAAS-67

### PR DESCRIPTION
**Pull request by vbudhram**
🎓 confirm password experiment

Fixes #5854

### CODE REVIEW

**Comment by vbudhram** [comment]
@mozilla/fxa-devs r?

File: app/scripts/templates/sign_up.mustache [hunk 1]
```diff
@@ -31,38 +31,27 @@
...
     <form novalidate>
-      <div class="input-row {{^forceEmail}}{{#showPasswordConfirm}}label-helper-top{{/showPasswordConfirm}}{{/forceEmail}}">
+      <div class="input-row {{^forceEmail}} label-helper-top{{/forceEmail}}">
         {{#forceEmail}}
           <p class="prefillEmail">{{ forceEmail }}</p>
           <input type="email" class="email hidden tooltip-below" value="{{ forceEmail }}" disabled />
         {{/forceEmail}}
         {{^forceEmail}}
-          {{#showPasswordConfirm}}
             <label class="label-helper"></label>
```
Comment (philbooth) [inline app/scripts/templates/sign_up.mustache]: Should we pull the indentation back to the same level as the `input`?

Comment (vladikoff) [reply app/scripts/templates/sign_up.mustache]: 👍

File: app/scripts/views/sign_up.js [hunk 2]
```diff
@@ -281,16 +279,8 @@ define(function (require, exports, module) {
...
-      return !! this.isInExperimentGroup('signupPasswordConfirm', 'treatment');
-    },
-
     _doPasswordsMatch() {
-      if (this._isPasswordConfirmEnabled()) {
-        return this.getElementValue('#password') === this.getElementValue('#vpassword');
-      } else {
-        return true;
-      }
+      return this.getElementValue('#password') === this.getElementValue('#vpassword');
```
Comment (vladikoff) [inline app/scripts/views/sign_up.js]: 👍

**Comment by vladikoff** [comment]
Just indent fix and ship it I think!

**Pull request merged**